modules/UI/side_pannels/profile/Profile.js

```diff
@@ -4,9 +4,8 @@ import UIEvents from '../../../../service/UI/UIEvents';
 import Settings from '../../../settings/Settings';
 
 import {
-    AUTHENTICATE_LOGIN_CLICKED,
-    AUTHENTICATE_LOGOUT_CLICKED,
-    sendAnalyticsEvent
+    createProfilePaneButtonEvent,
```

Comment:
Intentionally called Pane vs Panel? It is a pane but it's also referred to as a panel.

Reply:
I didn't know if there is a difference between "pane" and "panel" so I used the term that I saw in the documentation. But I'll change it to panel in order to avoid using an extra term.